.jenkins/cscs/env-clang-cuda.sh

```diff
@@ -29,6 +29,10 @@ configure_extra_options+=" -DHPX_WITH_CXX${CXX_STD}=ON"
 configure_extra_options+=" -DHPX_WITH_COMPILER_WARNINGS=ON"
 configure_extra_options+=" -DHPX_WITH_COMPILER_WARNINGS_AS_ERRORS=ON"
 configure_extra_options+=" -DHPX_WITH_SPINLOCK_DEADLOCK_DETECTION=ON"
+# Necessity to specify the cuda compiler when switching from FindCUDA to
+# enable_language(CUDA)
```

Comment:
Necessary?

Reply:
It fails currently I was just trying things but looks like it does't help on jenkins whereas it works locally, so any help appreciated ^^'

Reply:
Maybe I'm confused. I just meant if the comment is necessary? What fails?

Reply:
Ah understood will remove the comment :ok_hand:

Reply:
This fails:
https://cdash.cscs.ch/viewConfigure.php?buildid=157946